src/plugins/visualizations/common/constants.ts

```diff
@@ -31,3 +31,4 @@
  */
 
 export const VISUALIZE_ENABLE_LABS_SETTING = 'visualize:enableLabs';
+export const VISUALIZE_DISABLE_BUCKET_AGG_SETTING = 'visualize:disableBucketAggSettings';
```

Comment:
For consistency sake should it be `visualize:disableBucketAgg` ?

Reply:
sure, done